pyqtorch/utils.py

```diff
@@ -69,6 +71,44 @@ def overlap(bra: Tensor, ket: Tensor) -> Tensor:
     return torch.pow(inner_prod(bra, ket).real, 2)
 
 
+def sample_multinomial(
+    probs: Tensor,
+    length_bitstring: int,
+    n_samples: int,
+    return_counter: bool = True,
```

Comment:
Do we want to let the user decide on the return type ? It feels simpler and easier to maintain to agree on a single one.

Reply:
@Roland-djee I made it this way to factorize this function as `circuit.sample` would return the counter, and for the sampled expectation case with eigendecomposition, the tensor is needed for computation.